sound/soc/intel/common/soc-acpi-intel-ehl-match.c

```diff
@@ -8,8 +8,29 @@
 
 #include <sound/soc-acpi.h>
 #include <sound/soc-acpi-intel-match.h>
+#include "../skylake/skl.h"
+
+static struct skl_machine_pdata ehl_pdata = {
+	.use_tplg_pcm = true,
+};
 
 struct snd_soc_acpi_mach snd_soc_acpi_intel_ehl_machines[] = {
+	{
+		.id = "INT34C2",
+		.drv_name = "ehl_rt274",
+		.fw_filename = "intel/dsp_fw_ehl.bin",
+		.pdata = &ehl_pdata,
+		.sof_fw_filename = "sof-ehl.ri",
+		.sof_tplg_filename = "sof-ehl-rt274.tplg",
```

Comment:
Should this be sof-ehl-rt5660.tplg?

Reply:
"INT34C2" is not for ehl-rt5660 platform. This setting is from other teams.

Reply:
> "INT34C2" is not for ehl-rt5660 platform. This setting is from other teams.

It would be better not adding INT34C2 in this commit. We can add it in a separate commit or when ehl_rt274 machine driver is ready.

Reply:
OK, I will remove it. I added this because my code base is from IOTG, which has already included such codes.

Reply:
yes, don't add stuff we don't support.